test/Rentals.spec.ts

```diff
@@ -1338,6 +1431,81 @@ describe('Rentals', () => {
       expect(rental.endDate).to.equal(endDate)
     })
 
+    it('should allow calling accept offer twice if the second is an extension in the same block', async () => {
+      const rentalDays1 = 10
+      const rentalDays2 = 100
+
+      let rental = await rentals.rentals(listingParams.contractAddress, listingParams.tokenId)
```

Comment:
Can we check at this instance the lessor and tenant nonce?

Reply:
Checked on all similar extension in same block tests